github/pulls_reviews.go

```diff
@@ -17,3 +20,29 @@ type PullRequestReview struct {
 	// State can be "approved", "rejected", or "commented".
 	State *string `json:"state,omitempty"`
 }
+
+func (p PullRequestReview) String() string {
+	return Stringify(p)
+}
+
+// ListReviews lists all reviews on the specified pull request.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request
+func (s *PullRequestsService) ListReviews(owner string, repo string, number int) ([]*PullRequestReview, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pulls/%v/reviews", owner, repo, number)
```

Comment:
You can use "%d" instead of "%v" for number (but keep "%v" for owner and repo), in order to be more consistent with the rest of the code in this repository. See:

https://github.com/google/go-github/blob/466070b0580728e63bd1a415e0019639e55d7148/github/pulls.go#L212